src/main/resources/templates/openPrQueue.html

```diff
@@ -0,0 +1,42 @@
+{#include base}
+{#title}Open PR Queue{/title}
+
+{#body}
+<div class="ui main container">
+    <div class="ui icon message">
+        <i class="small sync icon"></i>
+        <div class="content">
+            <p>Updated {inject:prettyTime.format(organization.updated)} (refreshed every 10 minutes).</p>
+        </div>
+    </div>
+    {#for entry in organization.repositories}
+    <table class="ui celled striped table">
```

Comment:
Let's do this to be in synch with Monthly stats
```html
    <h2>{entry.value.size} open Pull Requests in {entry.key}</h2>
    <table class="ui celled striped table">
        <thead>
        <tr>
            <th>Created At</th>
            <th>Title</th>
            <th>Author</th>
            <th>Reviewers</th>
        </tr>
        </thead>
```

Reply:
Changed.